src/controllers/contract/types.ts

```diff
@@ -0,0 +1,19 @@
+export interface FindContractRequest {
+  id: string;
+}
+
+export interface CreateContractRequest {
+  beneficiary_id: string;
+  provider_id: string;
+  category_id: string;
+  start_date: Date;
```

Comment:
all are optional since we don't know what the client will request

Reply:
even tho they are all required in database, since we don't know how the user will request the API, we need to keep them as optional

Reply:
All optional now